fapolicyd.spec

```
@@ -94,21 +115,28 @@ sed -i "s|%ld_so_path%|`realpath $interpret`|g" rules.d/*.rules
 
 %build
 ./autogen.sh
-%configure \
-    --with-audit \
-#ELN    --with-rpm \
+configure_flags= --with-audit \
+    --with-rpm \
     --disable-shared
```

the --with-rpm should be conditional, I think

```suggestion
configure_flags= --with-audit \
    --disable-shared
    
%if %{defined eln_build}
configure_flags="$configure_flags --with-rpm"
%endif
```